pkg/ccl/testutilsccl/alter_primary_key.go

```diff
@@ -67,6 +67,11 @@ func AlterPrimaryKeyCorrectZoneConfigTest(
 			}
 
 			runCheck := false
+			sql.SecondaryTenantsMultiRegionAbstractionsEnabled.Override(
```

Comment:
nit: Consider adding a comment about why we're overriding this setting so we don't have to delve into git history to understand why?

Reply:
done